packages/gluestick/package.json

```diff
@@ -102,6 +102,7 @@
     "oy-vey": "0.9.0",
     "pm2": "2.3.0",
     "progress": "2.0.0",
+    "query-string": "^4.2.2",
```

Comment:
This doesn't affect our yarn.lock file because this is simply explicitly adding the query-string package that history uses